src/views/LoginView.tsx

```diff
@@ -70,6 +70,13 @@ export const LoginView = () => {
           // padding: '5%',
         }}
       >
+        <div style={{position:'relative', height:'12vh'}}>
```

Comment:
What's up with the height here?

Reply:
I am not sure this is going to scale. Does it look good on desktop?

Reply:
Styling is now fixed.